docs/tutorial/part-four/index.md

```diff
@@ -293,64 +293,74 @@ Page queries live outside of the component definition -- by convention at the en
 
 [StaticQuery](/docs/static-query/) is a new API introduced in Gatsby v2 that allows non-page components (like our `layout.js` component), to retrieve data via GraphQL queries.
 
-Go ahead and add a `<StaticQuery />` to your `src/components/layout.js` file and a `{data.site.siteMetadata.title}` reference that uses this data. When you are done your file looks like this:
+There are two ways to use StaticQuery:-
```

Comment:
I think it's really important when writing a tutorial to have an opinionated choice here. In other words--as a reader of this document new to the space, my first thought reading this will be one of confusion. _Why_ would I use one or the other? Why am I being given this choice?

So I'd prefer we're consistent here and only introduce one choice.

@marcysutton what do you think?

Reply:
That makes sense and I agree, good thinking.

I will stand by the thought that presenting `useStaticQuery` over `<StaticQuery>` is a good one though, because it feels neater and easier to understand coming at this new. But that might just be me and my colleague 😄

Reply:
Just want to chime in to say that you’ll need React and ReactDOM 16.8.0 or later to use `useStaticQuery`

We might want to mention that to avoid confusion if we switch this tutorial to recommend `useStaticQuery`

Reply:
I figured that because those following the tutorial would (presumably) be on the current version which already includes those it would confuse matters to mention it.